easybuild/easyconfigs/u/UCX/UCX-1.12.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'ConfigureMake'
+
+name = 'UCX'
+version = '1.12.1'
```

Comment:
@jfgrimm I've opened a separate PR for `UCX` in #15451 to already make some progress on this.

UCX is also required as a dependency for `impi`, so it's worth splitting this off imho.
Maybe we should also have a separate PR for `libfabric` + dependencies?

Reply:
dropped `UCX` from this PR, and moved `libfabric`, `libevent`, `PMIx` into #15456